fhir-persistence/src/test/resources/logging.unitTest.properties

```
@@ -35,7 +35,7 @@ handlers= java.util.logging.ConsoleHandler
 ############################################################
 
 # Limit the messages that are printed to the console
-java.util.logging.ConsoleHandler.level = FINE
+java.util.logging.ConsoleHandler.level = INFO
```

Given the other find/replace issue, I'm not sure if this change was intentional or not.  I think we should leave it at FINE so that you only need to change one place (below) to enable FINE logging to console.
```suggestion
java.util.logging.ConsoleHandler.level = FINE
```